README.md

```diff
@@ -1,450 +1,477 @@
+
+When you finish (your first attempt at) the task, create a Pull Request of your working branch into alpha. Also move your issue from the "In Progress" state into the "In Review" state. Communicate with the software Team and Team Lead, and your Team Lead will facilitate reviewing and approving the PR and placing the task into the "Done" state.
+
+
+## Branching
+
+### Strategy
+We have 5 "levels" of branching used:
+1. **master** -> This is reserved for FLIGHT READY code, i.e. very well tested.
+2. **beta**   -> This is reserved for FLATSAT code, i.e. moderately well tested.
+3. **alpha**  -> Working development branch, reserved for code that has been at least partially reviewed / tested.
+4. **other**  -> These branches are the only places where development happens. Each of these is based off of alpha.
+5. **hotfix** -> These are quick, one-off branches intended for quick fixes that are found on alpha or beta branches.
+
+Anyone can create a branch off of alpha and start developing. However, 2 people are required to review a PR (Pull Request) before the code can be accepted into the alpha branch. One of these people must be a team lead or project manager.
+
+alpha can be merged into beta, and beta can be merged into master. These are done very seldom throughout the project as the codebase matures, and may ONLY be done by the Software and Command Team Lead(s) or CubeSat Project Managers.  
+
+### Procedure
+1. In your local repo run ```git checkout alpha``` (you may have to commit, stash, or throw away uncommitted changes on your current branch)
+2. Run ```git pull``` (makes sure you have the latest code)
+2. run ```git checkout -b "<your branch name>"``` (creates a new branch)
+3. The first time that you try to push on the branch it will throw an error. Just follow the instructions to set the upstream branch.
+
+### Naming
+All branches **MUST** follow the few branch naming rules. Those rules are:
+- No captials
+- No underscores
+- Use hyphens instead of spaces
+- Must prepend new branch into one of six directories (see below)
+
+GitHub (and most other Git platforms) allow you to use branch folders, simply by uses forward slashes. Some examples of *good* branch names:
+- ```admin/restructure-directories```
+- ```test/write-unit-test-framework```
+- ```application/implement-payload-collection-task```
+- ```operation/import-nanopb```
+- ```framework/implement-uart-wrapper```
+- ```hotfix/fix-i2c-bug```
+
+Notice that all six of the directories used are based off of the Project names for the RADSAT-SK GitHub repo (minus hotfix, which is for quick fixes on alpha or beta branches).
+
+### Pull Request Procedure
+When bringing a new feature into flight code, you must merge your feature branch into the alpha branch after a review process know as Pull Requests or PR. To create a PR from your local machine:
+- Commit your local changes using ```git commit```

```

Comment:
Would it be beneficial to expand this to include adding files to be commited (e.g. `git add -A`), and/or adding commit messages (e.g. `git commit -am "refactored X"`)? Just a thought, for the sake of completeness